augur/api/view/routes.py

```diff
@@ -213,7 +188,95 @@ def user_delete():
 @app.route('/account/settings')
 @login_required
 def user_settings():
-    return render_template("settings.j2")
+    return render_template('settings.j2')
+
+
+@app.route('/account/api-keys')
+@login_required
+def api_keys_list():
```

Comment:
Endpoints returning a non-view response should not go in this file, all routes in this file must return a view. Routes returning JSON data or any other non-view components should be placed in either `augur.api.view.api` or `augur.api.routes`.